Unity/Assets/Scripts/json/JsonManager.cs

```diff
@@ -0,0 +1,119 @@
+using System;
+// using System.Collections;
+// using System.Collections.Generic;
+using UnityEngine;
+using System.IO;
+
+
+//class名要検討 どんなデータなのか分かる名前だとよい
+[Serializable]
+public class JsonData
```

Comment:
```suggestion
public class AppData
```
アプリに関するデータということで、ルートはAppDataでどうでしょうか